### ISSUE TITLE
Spammy build output

### ISSUE THREAD

**Issue by chetan** [habitat-sh/habitat#1280]
#1233 considerably improves the output of running `hab pkg build ...` by quieting the `hab pkg install` output during studio setup, but once the build actually starts, packages installed by `hab-plan-build.sh` output progress bars again. I've found the fix is to modify/override the following method:

``` bash
_install_dependency() {
  if [[ -z "${NO_INSTALL_DEPS:-}" ]]; then
    $HAB_BIN install -u $HAB_DEPOT_URL "$dep" || true
  fi
  return 0
}
```

from https://github.com/habitat-sh/habitat/blob/master/components/plan-build/bin/hab-plan-build.sh#L616-L621

the relevant line becomes:

```
    ($HAB_BIN install -u $HAB_DEPOT_URL "$dep" | cat -) || true
```

This line also needs `--verbose` removed:

```
 $_xz_cmd --compress -6 --threads=0 --verbose $tarf
```

https://github.com/habitat-sh/habitat/blob/master/components/plan-build/bin/hab-plan-build.sh#L2185

While this does work, there should probably be a better way of passing the correct behavior into this script. Perhaps something like wget's `--no-verbose`?

**Comment by chetan** [comment]
Did some more digging into this and discovered the following - 

* `hab pkg build` initially quiets output by requiring `isatty` on stdin/stdout/stderr
* however once inside the `chroot`ed env, I observe the following

```
$ hab pkg build . 2>&1 | cat - # redir/pipe to remove tty

$ ps auxwwwf | grep ..
root     15565  0.1  0.0  25176  3792 /hab/studios/testing/dev/pts/10 Ss+ 17:03   0:00          |               |               \_ /hab/pkgs/core/bash/4.3.42/20161213234235/bin/bash /hab/pkgs/core/hab-plan-build/0.15.0/20161222203825/bin/hab-plan-build .

$ ls -al /proc/15565/fd/
total 0
dr-x------ 2 root root  0 Jan 18 17:04 .
dr-xr-xr-x 9 root root  0 Jan 18 17:03 ..
lrwx------ 1 root root 64 Jan 18 17:04 0 -> /hab/studios/testing/dev/pts/10
lrwx------ 1 root root 64 Jan 18 17:04 1 -> /hab/studios/testing/dev/pts/10
lrwx------ 1 root root 64 Jan 18 17:04 2 -> /hab/studios/testing/dev/pts/10
```

So although our process initially disabled the tty, once inside the chroot env, we establish a new tty. I think the proper fix here is a probably an environment variable (`HAB_VERBOSE=false`?) that gets propagated into each subshell as necessary.

**Comment by smith** [comment]
Fixed in #1682. @chetan if this still doesn't do what you want, please reopen. Thanks!
